### ISSUE TITLE
Update column headings 

### ISSUE THREAD

**Issue by alisonrclarke** [DurhamARC/metalation-calculator#16]
Display a shorter version of the column heading, with a longer version becoming visible on hover.

See "Metalation calculator column labels.docx" in shared drive.

Consider using [MathJax](https://www.mathjax.org) to improve appearance of symbols, or see https://sites.psu.edu/symbolcodes/accents/math/#browser for other ideas.

Ensure the CSV export uses plain text headings.

**Comment by alisonrclarke** [comment]
Can you double-check the CSV before marking this as closed? Andrew had mentioned the delta symbols weren't appearing properly for him: it might just be a case of checking it's encoded properly.

**Comment by zeynepaki** [comment]
<img width="726" alt="Screenshot 2021-11-29 at 13 07 50" src="https://user-images.githubusercontent.com/73976500/143873567-8f87d9c5-4d89-44a0-a626-5d48e7672f3a.png">
Delta symbols appear fine to me upon download, should we do anything about the span content within the civ downloads?

**Comment by alisonrclarke** [comment]
It looks OK to me too but Andrew mentioned that it didn't work as expected for him; could you email him and ask him whether that's still the case, if he could email an exported file over, and what OS and tool he was using to look at the file?

Probably also worth asking him and Nigel whether he thinks the tooltip text should appear in the CSV or not?
